src/index.js

```diff
@@ -0,0 +1,23 @@
+const { ApolloServer, gql } = require("apollo-server");
+
+const typeDefs = gql`
+  type Query {
+    "A simple type for getting started!"
+    hello: String
+  }
+`;
+
+const resolvers = {
+  Query: {
+    hello: () => "world"
+  }
+};
+
+const server = new ApolloServer({
+  typeDefs,
+  resolvers
+});
+
+server.listen().then(({ url }) => {
```

Comment:
i would specify a port here. The post should pull from a .env file. This is a good opportunity to pull in dotenv. Its a lot easier to setup in node

Reply:
Done